src/components/PoolOverview.tsx

```diff
@@ -35,6 +35,15 @@ function PoolOverview({
           }
         : acc
     }, {} as Partial<Record<Partners, string>>),
+    apy: poolData.apy
+      ? String((100 * parseFloat(poolData.apy)).toFixed(1)) + "%"
+      : "",
+    volume: poolData.volume
+      ? "$" + commify(parseFloat(poolData.volume).toFixed(0))
+      : "",
+    utilization: poolData.utilization
```

Comment:
per the design, we can remove utilization from the `PoolOverview` and just include it in the `PoolInfoCard` component